app/src/main/res/values/strings.xml

```diff
@@ -47,7 +47,7 @@
     <string name="help_text_settings_for_testing">App Testers: Use the test server which will allow you to increase the recording frequency and use short recordings. Note: Data and recordings on the server are likely to be wiped at some point.</string>
     <string name="help_text_manage_recordings">Normally you will not need to \’Manage Recordings\’ as the app will automatically upload your recordings to the server.\n\nHowever, if you have been using the phone without an internet connection (perhaps you have been for a walk or have just re-established an internet connection after leaving your phone to make recordings in a location without out internet access) you can now use this screen to upload all the recordings on the phone.  \n\nBefore you press the upload button, the number of recordings currently on the phone is displayed and should change to zero if all the recordings are successfully uploaded.\n\nIf you prefer, you can delete all the recordings on the phone – perhaps you didn’t realize you had been making recordings! Careful with this option – you can’t get them back.</string>
     <string name="help_text_settings_for_audio_source">Take a look at: https://developer.android.com/reference/android/media/MediaRecorder.AudioSource</string>
-    <string name="help_text_settings_for_bird_count">The Bird Count feature aims to replicate the typical senario of sitting at a location and manually counting birds.  Choose the most appropriate duration and press the Record button.  \n\nNote: Once recording has finished, the app will attempt to upload the recordings - If you do not want this behaviour enable Offline mode in the app (See the Internet Connection screen under Advanced).</string>
+    <string name="help_text_settings_for_bird_count">The Bird Count feature aims to replicate the typical senario of sitting at a location and manually counting birds.\n\nWhen you do a normal 5 minute bird count you can record it with this application and it will send it to our database with GPS and time stamp so it can archived for future analysis.\n\nChoose the most appropriate duration and press the Record button.\n\nFor best results use an external microphone - https://www.2040.co.nz/.  \n\nNote: Once recording has finished, the app will attempt to upload the recordings - If you do not want this behaviour enable Offline mode in the app (See the Internet Connection screen under Advanced).</string>
```

Comment:
Suggested improvement to the last sentence: `If you don't want recordings to be automatically uploaded, enable Offline mode (see the Internet Connection screen under Advanced).`